src/tests/integration/test_questDB.py

```diff
@@ -546,21 +547,21 @@ def test_sqlcmd_not_supported_error(ip_questdb, query, capsys):
     [
         (
             lambda: (ggplot(penguins_data, aes(x="body_mass_g")) + geom_boxplot()),
-            f"boxplot {NOT_SUPPORTED_SUFFIX}",
+            "unbalanced",
         ),
         (
             lambda: (
                 ggplot(table="no_nulls", with_="no_nulls", mapping=aes(x="body_mass_g"))
                 + geom_boxplot()
             ),
-            f"boxplot {NOT_SUPPORTED_SUFFIX}",
+            "unbalanced",
         ),
     ],
 )
 def test_ggplot_boxplot_not_supported_error(
     ip_questdb, penguins_no_nulls_questdb, penguins_data, func, expected_error_message
 ):
-    with pytest.raises(UsageError) as err:
+    with pytest.raises(pg2.DatabaseError) as err:
```

Comment:
I'm not sure what this test does. are we testing boxplot with unbalanced data? if so, let's change the name of the test. also, shouldn't it raise `UsageError`?

Reply:
Removed the test, as discussed in monday meeting